google-cloud-video_intelligence/.yardopts

```diff
@@ -0,0 +1,8 @@
+--no-private
+--title=Google Cloud Video Intelligence API
+--exclude lib/google/cloud/video_intelligence/v1beta1
```

Comment:
I have locally fixed and tested the `.yardopts` exclusions, and fixed incorrect paths in `toc.json`. I am now looking into problems with the "Data Types" generation.